sci-libs/openblas/openblas2-0.2.20.recipe

```diff
@@ -1,22 +1,27 @@
 SUMMARY="An optimized BLAS library"
-DESCRIPTION="OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD\
- version."
+DESCRIPTION="OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD \
+version."
 HOMEPAGE="http://www.openblas.net/"
 COPYRIGHT="2011-2017 The OpenBLAS Project"
 LICENSE="BSD (3-clause)"
-REVISION="2"
+REVISION="3"
 SOURCE_URI="http://github.com/xianyi/OpenBLAS/archive/v$portVersion.tar.gz"
```

Comment:
`s/http/https/`

BTW, although I'm OK with the revision bump, I think other contributors might prefer a reset because we have "renames" in `PROVIDES` and `PROVIDES_devel`.

Also, are you sure openblas2 and openblas3 don't overlap?

Reply:
They overlap for sure. What should we do then?

Reply:
There are 2 solutions:

A) We just need to add a
```
CONFLICTS="
	openblas2$secondaryArchSuffix
	"
```
to `openblas3-0.3.1.recipe`.

Pros:
This would allow to have prebuilt packages for both openblas2 and openblas3.

Cons:
The "cost" for this is that we have two recipes, and that upgrades won't be automatic, unless we add a `REPLACES` to one of the recipes.

B) The other way to handle this is to drop openblas `0.2.x` and only keep `0.3.1`, in `openblas-0.3.1.recipe`, i.e. without the rename from "openblas" to "openblas3".
Pros:
No need for any `CONFLICTS{,_devel}` or `REPLACES`.
Cons:
`0.2.x` won't be available.

(I don't know which is best.)

Reply:
I rebuilt all the dependent packages with openblas3, there was no problems.
I'm not against dropping the old recipe.
@korli: any input?

Reply:
I usually like to drop the old version, but it seems like Haiku has [an interest in keeping old software running][0]. So I think we should keep the old version until security problems start to surface.

[0]: https://github.com/haikuports/haikuports/issues/2788#issuecomment-404202842